src/cljs/org/broadinstitute/firecloud_ui/common/sign_in.cljs

```diff
@@ -19,12 +19,21 @@
      [comps/Button
       {:text "Sign In"
        :onClick (fn [e]
-                  (.. js/window
-                      (open
-                       (str (config/api-url-root) "/login?callback="
-                            (js/encodeURIComponent (.. js/window -location -origin)))
-                       "Authentication"
-                       "menubar=no,toolbar=no,width=500,height=500")))}])
+                  (u/ajax {:url (str (config/api-url-root) "/me")
+                                  :on-done (fn [{:keys [success? status-code]}]
```

Comment:
We generally follow this style guide: https://github.com/bbatsov/clojure-style-guide

Please do a style pass through this PR. If your editor of choice doesn't automatically format the code correctly, I'm happy to come help configure it.